internal/util/rocksmq/server/rocksmq/rocksmq_reader.go

```diff
@@ -104,28 +104,7 @@ func (rr *rocksmqReader) Next(ctx context.Context) (*ConsumerMessage, error) {
 }
 
 func (rr *rocksmqReader) HasNext() bool {
-	if rr.iter.Valid() {
-		return true
-	}
-
-	select {
```

Comment:
I don't think the select can be removed, because when there are new kvs put into rocksdb, the old iteration won't know, the iterator needs to be regenerated,

Reply:
I will just revert the change for now

Reply:
just revert it for now